src/Microsoft.ApplicationInsights.AspNet/Config/TelemetryConfigurationSource.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet.Config
+{
+    using Microsoft.Framework.ConfigurationModel;
+    using System;
+    using System.Collections.Generic;
+
+    internal class TelemetryConfigurationSource : IConfigurationSource
+    {
+        /// <summary>
+        /// Developer mode setting value.
+        /// </summary>
+        public bool? DeveloperMode { get; set; }
+
+        /// <summary>
+        /// Instrumentation key setting value.
+        /// </summary>
+        public string InstrumentationKey { get; set; }
+
+        /// <summary>
```

Comment:
Doc comments in this file look autogenerated and don't add much value to this code. Since this is an internal class, can we remove them?